config/locales/server.en.yml

```diff
@@ -4677,6 +4679,10 @@ en:
           </ul>
           Popular theme components (for more, browse <a href='https://meta.discourse.org/c/theme/l/latest' target='_blank'>#theme</a>)"
 
+      fonts:
+        title: "Fonts"
+        description: "The base font used in most places of your site. This can be changed later and themes can override it."
```

Comment:
```suggestion
        description: "Choose a font that matches your theme or branding."
```

Reply:
My thinking here is that we don't tell them that other settings can be changed (they all can.) And themes seems a bit advanced at this point.

Reply:
Yes, I agree with you. I decided to remove this message completely and leave only the font previews, similar to how we do it on the themes page. 😊